viite-UI/src/view/navigationpanel/ProjectListModel.js

```diff
@@ -103,6 +105,11 @@
         });
       };
 
+      var cutProjectName = function (name) {
+        var maxNameLength = 24;
```

Comment:
Same in VIITE-1364 but different lengths? :/

Reply:
There was more space for the name here but I can change them both to be same.